docker/generic/start_proxy.py

```diff
@@ -869,6 +869,15 @@ def make_argparser():
         in grpc-json transcoding. This is to support HTML 2.0<https://tools.ietf.org/html/rfc1866#section-8.2.1>.
         Set this flag to true to disable this feature.
         ''')
+    parser.add_argument(
+        '--exclude_colon_in_url_wildcard_path_segment', action='store_true',
+        help='''
```

Comment:
Should we default this to true and let users set false manually? I know it is a potential breaking change, but it can lead mismatched route (and potential auth bypass) if we keep it disabled.

Reply:
@qiwzhang

Reply:
+ 1 on this idea. Done.